src/JSDomEnvironment.js

```diff
@@ -29,7 +31,8 @@ function JSDomEnvironment(config) {
   // use it (depending on the context -- such as TestRunner.js when operating as
   // a workerpool parent), this is the best way to ensure we only spend time
   // require()ing this when necessary.
-  this.global = require('./lib/jsdom-compat').jsdom().parentWindow;
+  this.global = require('./lib/jsdom-compat').jsdom().defaultView;
+  this.global.window = this.global;
```

Comment:
This looks problematic, afaik in jsdom we had problems with making sure `this === window` with some similiar constructs, are you sure that works?